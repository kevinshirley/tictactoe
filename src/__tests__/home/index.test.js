import React from 'react';
import { shallow } from 'enzyme';
import Home from '../../components/home';

describe('Home', () => {
  it('should render correctly in "debug" mode', () => {
    const home = shallow(<Home debug />);
  
    expect(home).toMatchSnapshot();
  });
});