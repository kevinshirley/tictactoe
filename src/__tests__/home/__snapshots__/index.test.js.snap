// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render correctly in "debug" mode 1`] = `
<section
  className="home"
>
  <div
    className="overlay"
  >
    <Title />
    <MainNav
      onNewGame={[Function]}
    />
    <Connect(withRouter(NewGameModal))
      isNewGameModalClose={false}
      isNewGameModalOpen={false}
      onReset={[Function]}
      quitModal={[Function]}
    />
  </div>
</section>
`;
